macros/src/main/scala/me/rexim/morganey/meta/QuotationMacro.scala

```diff
@@ -0,0 +1,161 @@
+package me.rexim.morganey.meta
+
+import me.rexim.morganey.ast._
+
+import scala.collection.mutable.ArrayBuffer
+import scala.reflect.macros.whitebox.Context
+
+private[meta] class QuotationMacro(val c: Context) {
```

Comment:
The macro project can of course be unit-tested separately, but I think It's impossible to test the macro-code itself (class QuotationMacro). As of now [SIP-5989](https://issues.scala-lang.org/browse/SI-5989) does still exist, which addresses the (unit-)testability of macros.